test/JDBC/upgrade/15_6/schedule

```diff
@@ -223,8 +223,6 @@ format-dep
 forxml
 forxml-subquery
 fulltextserviceproperty
-FULLTEXT_INDEX
-fts-contains
```

Comment:
If we are disabling this feature, let's ignore these tests in `jdbc_schedule` also. We would still have tests in `BABEL-UNSUPPORTED` to show that `FTS is not currently supported`.